django_cyverse_auth/views.py

```diff
@@ -195,7 +195,7 @@ def token_auth(request):
 
     if cas_validateUser(username):
         logger.info("CAS User %s validated. Creating auth token" % username)
-        token = createAuthToken(username)
+        token = create_token(username)
```

Comment:
The only choice considering the call signiture is:
 line 165 of models.py

def create_token(username, token_key=None, token_expire=None, remote_ip=None, issuer=None)